akka-bench-jmh/src/main/scala/akka/stream/MaterializationBenchmark.scala

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright (C) 2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+
+package akka.stream
+
+import java.util.concurrent.TimeUnit
+import akka.actor.ActorSystem
+import akka.stream.scaladsl._
+import org.openjdk.jmh.annotations._
+
+object MaterializationBenchmark {
+
+  val complexGraphBuilder = () =>
+    FlowGraph.closed() { implicit b =>
+      val numOfJunctions = 100
```

Comment:
Would be cool if `numOfJunctions` was a `@Param("10", "100", "1000")` to see how it grows